src/commonMain/kotlin/io/github/detekt/sarif4k/Merging.kt

```diff
@@ -0,0 +1,47 @@
+package io.github.detekt.sarif4k
```

Comment:
```suggestion
@file:JvmName("Merging")
package io.github.detekt.sarif4k
```
so it looks more natural from Java and Groovy? (when used in Gradle build scripts and plugins for example)